server/lib/nl/config_builder/builder.py

```diff
@@ -73,11 +74,24 @@ def build(state: PopulateState, config: Config) -> SubjectPageConfig:
   uttr.counters.timeit('get_sv_details', start)
 
   builder = base.Builder(uttr, sv2thing, config)
+  user_message = ''
+  for idx, cspec in enumerate(uttr.rankedCharts):
+    cspec = cast(ChartSpec, cspec)
+    if idx == 0:
+      user_message = cspec.info_message
+    elif cspec.info_message != user_message:
+      user_message = ''
+      break
 
   # Build chart blocks
   for cspec in uttr.rankedCharts:
     cspec = cast(ChartSpec, cspec)
     cv = cspec.chart_vars
+    # if there is a user message, that means every chart spec had the same info
+    # message and we should show the message at the page level instead of block
+    # level.
+    if bool(user_message):
```

Comment:
Curious what the `bool()` does over just `user_message`?

Reply:
I was using it to return false if user_message is empty, but an empty string itself will return false, so removed the `bool()`